src/components/Artists/SearchForm.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SearchForm = ({ searchTerm, onInputChange, onSearch }) => (
+  <section>
+    <form onSubmit={onSearch}>
```

Comment:
onSubmit right next to onSearch looks a little confusing to my eye, might be a good reason to pass these functions as handleSubmit or something similar.